examples/usecases/transformers-next-item-prediction-with-pretrained-embeddings.ipynb

```diff
@@ -0,0 +1,731 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    
</pre><p>I think we should not have two workflows.. we need to merge these two csv files first (you can do it with cudf, or pandas upfront, before NVT), and then apply categorify and other ops.. otherwise it will be wrongly mapped.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/1102/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
The way you describe it would be a good way to do it, but because of the high cardinality of the pretrained embeddings (50 in this case, but customers might likely encounter even larger sizes) our data would expand very significantly (we would need to add this information onto the 36+ million rows in the dataset).

This is only a WIP, I just wanted to see if I would be able to generate this dataset from a schema (was mostly concerned about the list columns that contain pretrained embedding information).

From what I understand that functionality is still in the works, but what we might be able to do is provide the train set along with information for pretrained embeddings separately to nvtabular/dataloader and the dataloder will be able to do the linking. 

And sorry, I just reread your comment - yes, you are spot on, that is a great observation! Yes, if this information were to be provided separately to two workflows, indeed the mapping after Categorify would be broken. But I don't believe we have the machinery available as of now to support providing both at the same time to Categorify (unless we do the merge like you describe, but this is not what we probably will want to do in the final example, once the functionality is fully implemented).